c_tests/trace_compiler/guard_eq.ll

```diff
@@ -0,0 +1,38 @@
+; Run-time:
+;   env-var: YKD_PRINT_IR=jit-pre-opt
+;   env-var: TRACE_DRIVER_BBS=main:0,main:1
+;   stderr:
+;      --- Begin jit-pre-opt ---
+;      ...
+;      define internal void @__yk_compiled_trace_0(...
+;        %{{0}} = alloca i32, align 4
+;        %{{1}} = icmp eq i32 1, 1
+;        br i1 %{{1}}, label %{{true}}, label %guardfail
+;
+;      guardfail:...
+;        call void (...) @llvm.experimental.deoptimize.isVoid(...
+;        ret void
+;
+;      {{true}}:...
+;        store i32 1, i32* %{{0}}, align 4
+;        ret void
+;      }
+;
+;      declare void @llvm.experimental.deoptimize.isVoid(...)
+;      --- End jit-pre-opt ---
+
+define void @main() {
+entry:
+    %0 = alloca i32
+    %1 = icmp eq i32 1, 1
+    br i1 %1, label %true, label %false
+
+true:
+    store i32 1, i32 * %0
+    unreachable
+
+false:
+    store i32 0, i32 * %0
+    unreachable
+
```

Comment:
Blank line?

Reply:
I think that's the implicit blank line vim inserts?

I don't see it in vim:
![image](https://user-images.githubusercontent.com/604955/153193607-2a5c9b70-d157-4024-b63a-fb94178c7794.png)

Reply:
oh wait :P

Reply:
Fixed in fb0a4a4

Reply:
"I don't see it in vim" :p